thoth/common/openshift.py

```diff
@@ -1404,6 +1405,9 @@ def schedule_adviser(
         if count is not None:
             template_parameters["THOTH_ADVISER_COUNT"] = str(count)
 
+        if from_package_update:
+            template_parameters["THOTH_ADVISER_DEPLOYMENT_PACKAGE_UPDATE"] = "1"
```

Comment:
Any reason not to put directly `THOTH_ADVISER_DEPLOYMENT_PACKAGE_UPDATE` - JSON describing the package to be updated?

Reply:
++, I realized this information will be present in Kebechet metadata. Should we just duplicate it here if the message type is package_release?

Reply:
Yes, I think so - adviser will need to be triggered with adjusted environment variables to run in the "update" mode. The function can accept an optional triplet (package name, package version, index url) that could be serialized to the env variable. The triplet can be supplied based on the message received.